### DISCUSSION TITLE
Tailwind UI

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#403] [General]
https://tailwindui.com/documentation

They've deprecated their Figma assets.
<img width="865" alt="Screenshot 2024-11-27 at 11 44 06" src="https://github.com/user-attachments/assets/a5a8d9a3-6aec-4e1d-8d7c-11f4efee86ac">
but they can still be downloaded.
https://tailwindui.com/download-figma

Maybe there are some bootleg Tailwind UI Figma files out there.

CC: @DenisMGojak @Happiartista @wintermeyer

**Comment by Happiartista** [comment]
These assets I've seen in the figma files are also quite limited @almirsarajcic @wintermeyer

**Comment by almirsarajcic** [comment]
This won't work as the Tailwind team has changed their workflow and doesn't use Figma anymore.
